osu.Game/Beatmaps/ControlPoints/DifficultyControlPoint.cs

```diff
@@ -29,6 +30,12 @@ public class DifficultyControlPoint : ControlPoint, IEquatable<DifficultyControl
             MaxValue = 10
         };
 
+        /// <summary>
+        /// Whether or not slider ticks should be generated at this control point.
+        /// This exists for backwards compatibility with maps that abuse NaN slider velocity behavior on osu!stable (e.g. /b/2628991).
+        /// </summary>
+        public readonly BindableBool GenerateTicksBindable = new BindableBool(true);
+
```

Comment:
Should not need to exist. Nothing is (nor should be) binding to this.

Reply:
I thought it was unnecessary too, but I wasn't passing tests because it didn't like `GenerateTicks` being used in `GetHashCode` due to something about that field not being read-only. There's probably a much, much better way of handling this but that's what I immediately jumped to.